src/libs/fetchPronouns.js

```diff
@@ -21,26 +23,24 @@ export async function fetchPronouns(dataID, accountName, type) {
 	debug(cacheResult);
 	// Extract the current cache by using object destructuring.
 	if (accountName in cacheResult.pronounsCache) {
-		const { value, timestamp } = cacheResult.pronounsCache[accountName];
+		const { value, timestamp, version } = cacheResult.pronounsCache[accountName];
```

Comment:
At this point we really shouldn't implement cache logic in this file anymore, otherwise it becomes way to messy once we want to support other pages.

Would be kind if you could extract it into the caching lib.